drivers/adc/adc_stm32.c

```diff
@@ -461,6 +461,10 @@ static int adc_stm32_read_async(const struct device *dev,
 
 static int adc_stm32_check_acq_time(uint16_t acq_time)
 {
+	if (acq_time == ADC_ACQ_TIME_MAX) {
+		return 7;
+	}
```

Comment:
maybe a slight comment  could explain why '7', where the value comes from

Reply:
@FRASTM 7 is maximal index (and represents longest time, as required in temperature conversion) in
```
static const uint16_t acq_time_tbl[8] = {2, 8, 14, 29, 42, 56, 72, 240};
```
Have no idea why `adc_stm32.c` uses magic numbers instead definitions.